BDArmory/VesselSpawning/SpawnUtils.cs

```diff
@@ -1141,18 +1145,56 @@ public void ApplyFJRT(Vessel vessel)
                     }
                 var AI = VesselModuleRegistry.GetModule<BDModulePilotAI>(vessel);
                 if (AI != null)
-                {
-                    if (BDArmorySettings.PS_EXTEND_DIST > 0) AI.extendTargetDist = Mathf.Min(AI.extendTargetDist, BDArmorySettings.PS_EXTEND_DIST);
-                    if (BDArmorySettings.PS_AVOID_THRESH > 0) AI.collisionAvoidanceThreshold = Mathf.Max(AI.collisionAvoidanceThreshold, BDArmorySettings.PS_AVOID_THRESH);
-                    if (BDArmorySettings.PS_AVOID_LA > 0) AI.vesselCollisionAvoidanceLookAheadPeriod = Mathf.Max(AI.vesselCollisionAvoidanceLookAheadPeriod, BDArmorySettings.PS_AVOID_LA);
-                    if (BDArmorySettings.PS_AVOID_STR > 0) AI.vesselCollisionAvoidanceStrength = Mathf.Max(AI.vesselCollisionAvoidanceStrength, BDArmorySettings.PS_AVOID_STR);
-                    if (BDArmorySettings.PS_IDLE_SPEED > 0) AI.idleSpeed = Mathf.Max(AI.idleSpeed, BDArmorySettings.PS_IDLE_SPEED);
+                {                    
+                    if (CompSettings.CompOverrides.ContainsKey("extendDistanceAirToAir"))
+                    {
+                        CompSettings.CompOverrides.TryGetValue("extendDistanceAirToAir", out float v);
```

Comment:
These should be written as
```csharp
if (CompSettings.CompOverrides.TryGetValue("extendDistanceAirToAir", out float v) && v >= 0)
    AI.extendDistanceAirToAir = Mathf.Min(AI.extendDistanceAirToAir, v);
```
otherwise, if `CompOverrides` doesn't contain a `"extendDistanceAirToAir"` key, then `v` will default to 0 and then be used for the override.
The enclosing `.ContainsKey` check then isn't necessary, it's essentially what the `TryGetValue` is doing.